packages/custom-webpack/README.md

```diff
@@ -216,6 +216,27 @@ new CircularDependencyPlugin({
 Keep in mind though that if there are default values in the plugin's constructor, they would override the corresponding values in the existing instance. So these you have to set explicitly to the same values Angular sets.  
 You can check out an example for plugins merge in the [unit tests](./src/webpack-config-merger.spec.ts) and in [this](https://github.com/meltedspark/angular-builders/issues/13) issue.
 
+## Custom Webpack promisified config
+
+Webpack config can also export a `Promise` object that resolves custom config. Given the following example:
+
+```js
+// extra-webpack.config.js
+const fs = require('fs');
+const util = require('util');
+const webpack = require('webpack');
+
+const readFile = util.promisify(fs.readFile);
+
+module.exports = readFile('./LICENSE', {
+  encoding: 'utf-8'
+}).then(license => ({
+  plugins: [new webpack.BannerPlugin(license)]
+}));
+```
```

Comment:
I think it is worth to add a word or two about support of async config function as well (maybe in Cstuom webpack config function section)